clients/python/kh_client.py

```diff
@@ -0,0 +1,61 @@
+import dataclasses
+import json
+import os
+from dataclasses import dataclass
+from typing import List
+
+import requests
```

Comment:
Could this package be included in a `requirements.txt` file -- I think this would help with installing dependencies for this package such as requests.